pandas/core/base.py

```diff
@@ -638,9 +638,7 @@ def _is_builtin_func(self, arg):
 
 
 class IndexOpsMixin:
-    """ common ops mixin to support a unified interface / docs for Series /
-    Index
-    """
+    """\nCommon ops mixin to support a unified interface / docs for Series / Index\n"""
```

Comment:
Any reason to use the newline character here instead of actually putting on a new line?

Reply:
Not sure. Resolved this.